rustup-init.sh

```diff
@@ -493,15 +493,27 @@ check_help_for() {
 
     case "$_arch" in
 
-        # If we're running on OS-X, older than 10.13, then we always
-        # fail to find these options to force fallback
         *darwin*)
         if check_cmd sw_vers; then
-            if [ "$(sw_vers -productVersion | cut -d. -f2)" -lt 13 ]; then
-                # Older than 10.13
-                echo "Warning: Detected OS X platform older than 10.13"
-                return 1
-            fi
+            case $(sw_vers -productVersion) in
+                10.*)
+                    # If we're running on OS-X, older than 10.13, then we always
```

Comment:
Being pedantic...

- "Mac OS X" 10.0 
- "OS X" 10.8
- "macOS" 10.12

"OS-X" was never a proper name. Since rustup stops at 10.13, I'd suggest changing to "macOS" consistently in user-facing text as well as comments.

Reply:
That's fair.  I shall make that tweak.